.ci/azure/pypi.yml

```diff
@@ -0,0 +1,72 @@
+jobs:
```

Comment:
Should merge these two jobs into one and then make the two deploy steps conditional

Reply:
I've merged the two deployment jobs into a single one with conditional steps. What I also did is split the build and the deployment into different jobs. And now we can build on every push and check if the build is correct.